### PR TITLE
Fixed GET api/3.1/cdns/routing endpoint to avoid incorrect success response (return all zeros) 

### PR DESCRIPTION
**Pull request by rimashah25**
## What does this PR (Pull Request) do?

- [x] This PR fixes a bug found during ATC v5.1 Release Validation 

## Which Traffic Control components are affected by this PR?

- Traffic Ops
- Traffic Router

## What is the best way to verify this PR?

Test GET API endpoint: api/3.1/cdns/routing against CIAB or local TR and if TR is not up an running or has any other issue, one should see the right error response and http code.

## If this is a bug fix, what versions of Traffic Control are affected?

- 5.1.1
- master (7f5c7be)

## The following criteria are ALL met by this PR

- [x] This PR does not includes tests.
- [x] I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md
- [x] This PR includes any and all required license headers
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

## Additional Information

Logic didn't change, hence there was no test or documentation added.

### CODE REVIEW

File: traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/StatTracker.java [hunk 1]
```diff
@@ -339,9 +339,9 @@ public void setTotalDsMissCount(final int totalDsMissCount) {
 		this.totalDsMissCount = totalDsMissCount;
 	}
 
-	private int totalDnsCount;
+	private long totalDnsCount;
 	private long totalDnsTime;
-	private int totalHttpCount;
+	private long totalHttpCount;
 	private long totalHttpTime;
 	private int totalDsMissCount = 0;
```
Comment (rawlinp) [inline traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/StatTracker.java:346]: We might as well make this a `long` as well just to round them all out.

**Review by mattjackson220 — APPROVED** [review]
Looks good to me! tests pass and it should fix the issue! nice!

**Review by zrhoffman — CHANGES_REQUESTED** [review]
> Fixed GET api/3.1/cdns/routing endpoint

Since it gets written to the git history, would you please edit the title to describe what issue with the `/api/3.1/cdns/routing` endpoint is fixed by this PR?

**Review by rawlinp — APPROVED** [review]
LGTM once more details are included in the title as @zrhoffman suggests.

**Review by zrhoffman — APPROVED** [review]

**Pull request merged**